src/test/java/hw6/steps/WhenSteps.java

```diff
@@ -0,0 +1,36 @@
+package hw6.steps;
+
+import hw3.pages.DifferentElementsPage;
+import hw3.pages.UserTablePage;
+import io.cucumber.java.en.When;
+import utils.WebDriverSingleton;
+
+public class WhenSteps {
+
+    private DifferentElementsPage differentElementsPage;
+    private UserTablePage userTablePage;
+
+    @When("I select checkboxes {string}, {string}")
+    public void iSelectCheckboxesOnDifferentElementsPage(String firstCheckbox, String secondCheckbox) {
+        differentElementsPage = new DifferentElementsPage(WebDriverSingleton.INSTANCE.getDriver());
+        differentElementsPage.setCheckBox(firstCheckbox);
+        differentElementsPage.setCheckBox(secondCheckbox);
+    }
```

Comment:
How will you manage if you will need check only one checkbox?

Reply:
added list